howto/monitor/collect-metrics.md

```diff
@@ -0,0 +1,227 @@
+This example implementation provides a starting point for a monitoring stack that can be used to gather metrics with [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) and [Prometheus](https://prometheus.io/) and access them through [Grafana](https://grafana.com/).
+
+In this setup, every LXD cluster node runs a Telegraf instance that gathers the machine metrics. All Anbox containers that exist on the node also report their metrics to the Telegraf instance.
+
+Prometheus gathers the metrics that are collected by the different Telegraf instances and combines them with the metrics provided directly by AMS. You can access, query and visualise the full metrics data through Grafana.
```

Comment:
There is not a combination of the metrics being done by Prometheus. It simply stores all metrics in its time series database which then can be queried.

All of our services (AMS, Anbox Stream Gateway, NATS) provide  prometheus endpoints

Reply:
But Prometheus picks up the data from different sources - correct?
That is what I was trying to say, will try to reword.

Reply:
Correct, prometheus scrapes metrics from different endpoints but they all go as individual records in the time series db Prometheus maintains.